po/de.po

```diff
@@ -10531,7 +10558,7 @@ msgstr "Rechte Maustaste"
 
 #: src/frontend.cpp:1453
 msgid "Map/Radar Zoom"
-msgstr "Karten-/Minikartenzoom"
+msgstr "(Mini)Kartenzoom"
```

Comment:
Das ist eine sehr unschöne Variante (vermutlich noch dazu nicht von den üblichen Rechtschreibregeln gedeckt), weshalb ich diese Änderung ablehne.

Reply:
War eine Idee, um den Begriff zu verkürzen. Forgon mag es nun mal nicht, wenn Begriffe über den Rand der GUI reichen. Belassen wir es bei der alten Variante.

Reply:
Begriffe, die über die GUI hinausreichen, sind tatsächlich ein Ärgernis. So ein Problem muss aber technisch gelöst werden, beispielsweise durch Nutzung einer Abkürzung oder gar nur eines Symbols, das beim Hovern dann den dazugehörigen Text in einer Box anzeigt.